README.md

```diff
@@ -5,21 +5,15 @@ examples:
 
 RP2040:
 
-```
-python3 parser.py "Adafruit Feather RP2040.fzpz" C:\Users\ladyada\Dropbox\micropython\circuitpython\ports\raspberrypi\boards\adafruit_feather_rp2040\pins.c rp2040pins.csv
-```
+```python3 parser.py "Adafruit Feather RP2040.fzpz" ~/Git/circuitpython/ports/raspberrypi/boards/adafruit_feather_rp2040/pins.c rp2040pins.csv```
```

Comment:
Please make the path more generalised. Consider the following.
```suggestion
```python3 parser.py "Adafruit Feather RP2040.fzpz" path/to/circuitpython/ports/raspberrypi/boards/adafruit_feather_rp2040/pins.c rp2040pins.csv```
```

Reply:
That looks better.   They all match now. This meant there was a duplicate entry for the Metro ESP32-S2.  Would you rather keep one with an example Windows file system or have them all line up?

Reply:
I missed this question in my review. I'd rather they all match up. If you want to remove the duplicate, that would be great.